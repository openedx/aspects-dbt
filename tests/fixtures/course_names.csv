course_key,display_name,course_run,org
course-v1:OEX+ecdf7b+2024-1,Course ecdf7b,2024-1,OEX
course-v1:OEX+0238f7+2024-1,Course 0238f7,2024-1,OEX
course-v1:OEX+071354+2024-1,Course 071354,2024-1,OEX
course-v1:OEX+0733ef+2024-1,Course 0733ef,2024-1,OEX
course-v1:OEX+08df8f+2024-1,Course 08df8f,2024-1,OEX
course-v1:OEX+095812+2024-1,Course 095812,2024-1,OEX
course-v1:OEX+09afd1+2024-1,Course 09afd1,2024-1,OEX
course-v1:OEX+0a9f76+2024-1,Course 0a9f76,2024-1,OEX
course-v1:OEX+0c4ae9+2024-1,Course 0c4ae9,2024-1,OEX
course-v1:OEX+0e6696+2024-1,Course 0e6696,2024-1,OEX
course-v1:OEX+171e4b+2024-1,Course 171e4b,2024-1,OEX
course-v1:OEX+1927a4+2024-1,Course 1927a4,2024-1,OEX
course-v1:OEX+20be6e+2024-1,Course 20be6e,2024-1,OEX
course-v1:OEX+211319+2024-1,Course 211319,2024-1,OEX
course-v1:OEX+21bc59+2024-1,Course 21bc59,2024-1,OEX
course-v1:OEX+25401e+2024-1,Course 25401e,2024-1,OEX
course-v1:OEX+3c19fd+2024-1,Course 3c19fd,2024-1,OEX
course-v1:OEX+3cc0e0+2024-1,Course 3cc0e0,2024-1,OEX
course-v1:OEX+411d40+2024-1,Course 411d40,2024-1,OEX
course-v1:OEX+42eece+2024-1,Course 42eece,2024-1,OEX
course-v1:OEX+43986c+2024-1,Course 43986c,2024-1,OEX
course-v1:OEX+494b9b+2024-1,Course 494b9b,2024-1,OEX
course-v1:OEX+506d83+2024-1,Course 506d83,2024-1,OEX
course-v1:OEX+569880+2024-1,Course 569880,2024-1,OEX
course-v1:OEX+59f6bf+2024-1,Course 59f6bf,2024-1,OEX
course-v1:OEX+5a62b9+2024-1,Course 5a62b9,2024-1,OEX
course-v1:OEX+61fc07+2024-1,Course 61fc07,2024-1,OEX
course-v1:OEX+6796cd+2024-1,Course 6796cd,2024-1,OEX
course-v1:OEX+695908+2024-1,Course 695908,2024-1,OEX
course-v1:OEX+6c0e3e+2024-1,Course 6c0e3e,2024-1,OEX
course-v1:OEX+6f7d2c+2024-1,Course 6f7d2c,2024-1,OEX
course-v1:OEX+75668b+2024-1,Course 75668b,2024-1,OEX
course-v1:OEX+7df30b+2024-1,Course 7df30b,2024-1,OEX
course-v1:OEX+7f013e+2024-1,Course 7f013e,2024-1,OEX
course-v1:OEX+83220c+2024-1,Course 83220c,2024-1,OEX
course-v1:OEX+86c912+2024-1,Course 86c912,2024-1,OEX
course-v1:OEX+87d959+2024-1,Course 87d959,2024-1,OEX
course-v1:OEX+90d0f4+2024-1,Course 90d0f4,2024-1,OEX
course-v1:OEX+95a04c+2024-1,Course 95a04c,2024-1,OEX
course-v1:OEX+9880cc+2024-1,Course 9880cc,2024-1,OEX
course-v1:OEX+9a79cf+2024-1,Course 9a79cf,2024-1,OEX
course-v1:OEX+9a8c6e+2024-1,Course 9a8c6e,2024-1,OEX
course-v1:OEX+a542a7+2024-1,Course a542a7,2024-1,OEX
course-v1:OEX+aece71+2024-1,Course aece71,2024-1,OEX
course-v1:OEX+b0698e+2024-1,Course b0698e,2024-1,OEX
course-v1:OEX+b6051b+2024-1,Course b6051b,2024-1,OEX
course-v1:OEX+bb741d+2024-1,Course bb741d,2024-1,OEX
course-v1:OEX+bb8001+2024-1,Course bb8001,2024-1,OEX
course-v1:OEX+c4f493+2024-1,Course c4f493,2024-1,OEX
course-v1:OEX+c52432+2024-1,Course c52432,2024-1,OEX
course-v1:OEX+c72f3a+2024-1,Course c72f3a,2024-1,OEX
course-v1:OEX+c85c20+2024-1,Course c85c20,2024-1,OEX
course-v1:OEX+cc5f70+2024-1,Course cc5f70,2024-1,OEX
course-v1:OEX+cda396+2024-1,Course cda396,2024-1,OEX
course-v1:OEX+cf4c1e+2024-1,Course cf4c1e,2024-1,OEX
course-v1:OEX+d52fd3+2024-1,Course d52fd3,2024-1,OEX
course-v1:OEX+d91324+2024-1,Course d91324,2024-1,OEX
course-v1:OEX+d94697+2024-1,Course d94697,2024-1,OEX
course-v1:OEX+dc9411+2024-1,Course dc9411,2024-1,OEX
course-v1:OEX+dece1e+2024-1,Course dece1e,2024-1,OEX
course-v1:OEX+df6d79+2024-1,Course df6d79,2024-1,OEX
course-v1:OEX+e9678d+2024-1,Course e9678d,2024-1,OEX
course-v1:OEX+ec8595+2024-1,Course ec8595,2024-1,OEX
course-v1:OEX+f029f9+2024-1,Course f029f9,2024-1,OEX
course-v1:OEX+f35f75+2024-1,Course f35f75,2024-1,OEX
course-v1:OEX+f563ab+2024-1,Course f563ab,2024-1,OEX
course-v1:OEX+f7a535+2024-1,Course f7a535,2024-1,OEX
course-v1:OEX+f8431d+2024-1,Course f8431d,2024-1,OEX
course-v1:OEX+f8a230+2024-1,Course f8a230,2024-1,OEX
course-v1:OEX+fa9921+2024-1,Course fa9921,2024-1,OEX
course-v1:OEX+fa9a50+2024-1,Course fa9a50,2024-1,OEX
course-v1:OEX+fc32c1+2024-1,Course fc32c1,2024-1,OEX
course-v1:OEX+fc6cb0+2024-1,Course fc6cb0,2024-1,OEX
course-v1:OEX+fcacfb+2024-1,Course fcacfb,2024-1,OEX
course-v1:OEX+fdcda6+2024-1,Course fdcda6,2024-1,OEX
course-v1:OEX+ff8b21+2024-1,Course ff8b21,2024-1,OEX
course-v1:OpenedX+DemoX+DemoCourse,Open edX Demo Course,DemoCourse,OpenedX
course-v1:Org0+DemoX+7b3f3a,7b3f3a (huge),7b3f3a,Org0
course-v1:Org0+DemoX+8f69aa,8f69aa (medium),8f69aa,Org0
course-v1:Org0+DemoX+a8dcc4,a8dcc4 (large),a8dcc4,Org0
course-v1:Org0+DemoX+a958e0,a958e0 (large),a958e0,Org0
course-v1:Org0+DemoX+ccdab1,ccdab1 (medium),ccdab1,Org0
course-v1:Org0+DemoX+e13303,e13303 (huge),e13303,Org0
course-v1:Org0+DemoX+efcc66,efcc66 (small),efcc66,Org0
course-v1:Org0+DemoX+f5e757,f5e757 (large),f5e757,Org0
course-v1:Org0+DemoX+f9520a,f9520a (huge),f9520a,Org0
course-v1:Org0+DemoX+fe5376,fe5376 (large),fe5376,Org0
course-v1:Org1+DemoX+1a3d9c,1a3d9c (huge),1a3d9c,Org1
course-v1:Org1+DemoX+486f9e,486f9e (huge),486f9e,Org1
course-v1:Org1+DemoX+9b6a1c,9b6a1c (huge),9b6a1c,Org1
course-v1:Org1+DemoX+a6f53f,a6f53f (small),a6f53f,Org1
course-v1:Org1+DemoX+a81e3b,a81e3b (medium),a81e3b,Org1
course-v1:Org1+DemoX+ba73c0,ba73c0 (small),ba73c0,Org1
course-v1:Org1+DemoX+e451fd,e451fd (medium),e451fd,Org1
course-v1:Org1+DemoX+ee5172,ee5172 (small),ee5172,Org1
course-v1:Org1+DemoX+fcb758,fcb758 (large),fcb758,Org1
course-v1:Org2+DemoX+2d7956,2d7956 (small),2d7956,Org2
course-v1:Org2+DemoX+56d401,56d401 (small),56d401,Org2
course-v1:Org2+DemoX+58af07,58af07 (huge),58af07,Org2
course-v1:Org2+DemoX+7238db,7238db (medium),7238db,Org2
course-v1:Org2+DemoX+778e7c,778e7c (medium),778e7c,Org2
course-v1:Org2+DemoX+ab8996,ab8996 (large),ab8996,Org2
course-v1:Org2+DemoX+d61648,d61648 (large),d61648,Org2
course-v1:Org2+DemoX+d89c8f,d89c8f (medium),d89c8f,Org2
course-v1:Org2+DemoX+fac9ba,fac9ba (small),fac9ba,Org2
