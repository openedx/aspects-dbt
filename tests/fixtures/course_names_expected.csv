"course_key","course_name","course_run","org"
"course-v1:Org0+DemoX+2bc51b","2bc51b (small)","2bc51b","Org0"
"course-v1:Org0+DemoX+81bba1","81bba1 (small)","81bba1","Org0"
"course-v1:Org3+DemoX+528fdd","528fdd (small)","528fdd","Org3"
"course-v1:Org2+DemoX+682526","682526 (medium)","682526","Org2"
"course-v1:Org7+DemoX+57295b","57295b (medium)","57295b","Org7"
"course-v1:Org8+DemoX+3fefec","3fefec (medium)","3fefec","Org8"
"course-v1:Org4+DemoX+db4c73","db4c73 (large)","db4c73","Org4"
"course-v1:Org1+DemoX+1937e7","1937e7 (large)","1937e7","Org1"
"course-v1:Org4+DemoX+0b1656","0b1656 (huge)","0b1656","Org4"
"course-v1:Org2+DemoX+e4380c","e4380c (huge)","e4380c","Org2"